sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/theme/AppColors.kt

```diff
@@ -0,0 +1,29 @@
+package co.nimblehq.sample.compose.ui.theme
+
+import androidx.compose.material.*
+import androidx.compose.runtime.staticCompositionLocalOf
+
+/**
+ * Extend final [Colors] class to provide more custom app colors.
+ */
+data class AppColors(
+    val themeColors: Colors,
+
+    // Custom colors here
+)
+
+internal val LightColorPalette = AppColors(
+    themeColors = lightColors(
+        primary = GreenCitrus,
+        secondary = GreenChristi
+    )
+)
+
+internal val DarkColorPalette = AppColors(
+    themeColors = darkColors(
+        primary = GreenCitrus,
```

Comment:
I think we can remove `primary` for `darkTheme`, same in `template-compose`.
```suggestion
```

Reply:
Yes, we don't need to specify `primary` and `secondary` for template-compose ✅

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/376/commits/5c66234ffe3de57f67c29970aa4554db2126aa48